opentelemetry-zipkin/Cargo.toml

```diff
@@ -41,9 +40,10 @@ thiserror = { workspace = true }
 futures-core = { workspace = true }
 
 [dev-dependencies]
+async-trait = { workspace = true }
 bytes = { workspace = true }
 futures-util = { workspace = true, features = ["io"] }
 http-body-util = { workspace = true }
 hyper-util = { workspace = true, features = ["client-legacy", "http1", "tokio"] }
-opentelemetry_sdk = { default-features = false, features = ["trace", "testing"], path = "../opentelemetry-sdk" }
+opentelemetry_sdk = { default-features = false, features = ["experimental_trace_batch_span_processor_with_async_runtime", "trace", "testing"], path = "../opentelemetry-sdk" }
```

Comment:
no need to use the experimental_batch_processor_with-asyc-runtime feature. This should work without that.